ext/pycall/pycall.c

```diff
@@ -1232,6 +1241,7 @@ pycall_libpython_helpers_m_dict_contains(VALUE mod, VALUE pyptr, VALUE key)
   if (res == -1) {
     pycall_pyerror_fetch_and_raise("PyDict_Contains");
   }
+  pycall_Py_DecRef(pyobj_key);
```

Comment:
Could you please move this `DecRef` just after calling `PyDict_Contains`?

Reply:
OK